fiat-go/32/poly1305/poly1305.go

```diff
@@ -30,6 +30,16 @@ package poly1305
 type uint1 uint8
 type int1 int8
 
+// The type LooseFieldElement is a field element with loose bounds.
+//
+// Bounds: [[0x0 ~> 0xc000000], [0x0 ~> 0xc000000], [0x0 ~> 0xc000000], [0x0 ~> 0xc000000], [0x0 ~> 0xc000000]]
```

Comment:
```suggestion
// Bounds:
//
//   [[0x0 ~> 0xc000000], [0x0 ~> 0xc000000], [0x0 ~> 0xc000000], [0x0 ~> 0xc000000], [0x0 ~> 0xc000000]]
```

Making it a code block would improve readability.

Reply:
Done